services/queries/ProjectQueryBuilder.js

```diff
@@ -109,6 +115,11 @@ class ProjectQueryBuilder {
     if (this.filters.userId) conditions.push(`p."userId" = :userId`);
     if (this.filters.status) conditions.push(`p."status" = :status`);
     if (this.filters.types?.length) conditions.push(`t."id" IN (:types)`);
+    if (this.filters.query) {
+      const query = this.filters.query.replace(/'/g, "''");
+      conditions.push(`(p."name" ILIKE '%${query}%')`);
```

Comment:
I see query is added to the replacements object, but the SQL is using string interpolation. 
Maybe change to parameterized queries and update the replacement value for better security/SQL injection protection.
```suggestion
      conditions.push(`(p."name" ILIKE :query)`);
```